spanner/transaction_test.go

```diff
@@ -413,19 +413,31 @@ func TestReadWriteTransaction_PrecommitToken(t *testing.T) {
 		query                  bool
 		update                 bool
 		batchUpdate            bool
+		mutationsOnly          bool
```

Comment:
Can you add one test case with client.Apply and verify the same behavior?
https://cloud.google.com/spanner/docs/modify-mutation-api#go